### ISSUE TITLE
How can I get the confidence score for the classification task

### ISSUE THREAD

**Issue by pinklady0120** [huggingface/pytorch-pretrained-BERT#219]
In evaluation step, it seems it only shows the predicted label for the data instance.
How can I get the confidence score for each class?

**Comment by rodgzilla** [comment]
You can use `torch.nn.functional.softmax` on the `logits` that the model outputs here:

https://github.com/huggingface/pytorch-pretrained-BERT/blob/0a9d7c7edb20a3e82cfbb4b72515575543784823/examples/run_classifier.py#L589-L591

It will give you the confidence score for each class.
